src/main/scala/com.snowplowanalytics/forex/Forex.scala

```diff
@@ -66,59 +64,39 @@ object Forex {
  * They return ForexLookupTo object which is then passed to methods
  * in ForexLookupTo class in the fluent interface.
  * @param config - a configurator for Forex object
- * @param clientConfig - a configurator for Forex Client object
- * @param nowishCache - user defined nowishCache
- * @param eodCache - user defined eodCache
+ * @param client - Passed down client that does actual work
```

Comment:
I see quite uncommon practice here with hashes between param name and its desription.